test/functional/feature_rbf.py

```diff
@@ -36,7 +37,7 @@ def set_test_params(self):
             [
                 "-acceptnonstdtxn=1",
                 "-maxorphantx=1000",
-                "-limitancestorcount=50",
+                f"-limitancestorcount={MAX_REPLACEMENT_LIMIT + 3}", # enough room to test BIP125 Rule #5
```

Comment:
Side note: It's not very clean to change a policy in order to test another, tightly related policy. If the test requires more than 100 replaced transactions, you should create multiple conflicting transactions rather than increasing the ancestor limit.